### DISCUSSION TITLE
Please Help - wrong $queryString on more updated components at once $refresh

### DISCUSSION THREAD

**Discussion by jirapetr** [livewire/livewire#5144] [1. Help]
I have 3 livewire components (A-component, B-component, Reset-component)
-----------------
**A-component:**

protected $listeners = ['deleteAllFilters' => '$refresh'];
protected $queryString = ['year'];
public $year;

------------------
**B-component:**

protected $listeners = ['deleteAllFilters' => '$refresh'];
protected $queryString = ['height'];
public height;

-----------------
**Reset-component:**

$this->emit('deleteAllFilters')

-----------------
Problem is that emited $refresh on B-component donot accept changes in URL 
from $refresh A-component (which is emited before) and year stayed in URL

(I also tried emit $this->reset('year') + $this->reset('height') separatly but with the same result

**Comment by joshhanley** [comment]
@jirapetr please see @Qirel's response to your message on Discord 🙂 https://discord.com/channels/698229985755791471/698231501501890600/1003627029192777778
